great_expectations/validator/validation_graph.py

```diff
@@ -71,15 +71,22 @@ def __init__(
 
         self._edge_ids = {edge.id for edge in self._edges}
 
+    def __eq__(self, other) -> bool:
+        """Supports comparing two "ValidationGraph" objects."""
+        return self.edge_ids == other.edge_ids
+
     @property
-    def edges(self):
+    def edges(self) -> List[MetricEdge]:
+        """Returns "MetricEdge" objects, contained within this "ValidationGraph" object (as list)."""
         return self._edges
 
     @property
-    def edge_ids(self):
+    def edge_ids(self) -> Set[Tuple[str, str]]:
```

Comment:
Can we get rid of _edge_ids and always use this instead. It looks like they are both derived from self._edges and we just keep them in sync on lines 91-92.

Reply:
@billdirks I will see what I can do (this code was written before I worked with it, so may not succeed if these IDs are used somewhere else).